native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -21,21 +22,26 @@ class ViewController: UIViewController, ApplicationContractView {
         self.arrivalPicker.delegate = self
         self.arrivalPicker.dataSource = self
         setupTableView()
+        tableView.alpha = 0
```

Comment:
What's the intention behind this line? Are you trying to hide the table?

Reply:
I believe so, I've changed so now use `tableView.isHidden = true` like is used elsewhere

Reply:
Good plan, that's a bit clearer to anyone reading the code 🙂